test/Microsoft.AspNet.Razor.Test/Parser/CSharp/CSharpBlockTest.cs

```diff
@@ -736,6 +737,231 @@ public void ParsersCanNestRecursively()
                     Factory.Code("        }").AsStatement().Accepts(AcceptedCharacters.None)));
         }
 
+        public static TheoryData ParseBlockData
```

Comment:
More specific name. `ParseBlockData` is to generic, you're testing the escaping of `@` symbols in attribute values (inside C# blocks).